src/httpledstripe_esp.cpp

```diff
@@ -1276,7 +1277,8 @@ void handleGetModes(AsyncWebServerRequest *request)
 {
   // will return all available effects in JSON as name, number 
 
-  AsyncJsonResponse * response = new AsyncJsonResponse();
+  // Buffer size: ~164 modes * 25 chars average (name + JSON structure) = ~4100 bytes + overhead = 5120 bytes
```

Comment:
At the moment, there are like 54 modes. I would count with 70 max and some overhead to land at 2048 bytes.